db/seeds.rb

```diff
@@ -0,0 +1,48 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+
+User.create(name: 'Hammas', email: 'hammas@gmail.com', password: 'hammas')
```

Comment:
- [ ] It looks like the seeds could be changed a little bit to work better, remember to create in a good way to create all seeds on the application. 🎧 (The issue could be on model validations and these seeds).
<img width="990" alt="image" src="https://user-images.githubusercontent.com/20563252/176568168-2c909fb8-7547-4f0c-b500-b8de1d801b20.png">